be/src/exec/orc_scanner.cpp

```diff
@@ -86,7 +86,7 @@ Status ORCScanner::open() {
     _orc_reader->set_timezone(_state->timezone());
     _orc_reader->drop_nanoseconds_in_datetime();
     _orc_reader->set_runtime_state(_state);
-    _orc_reader->set_case_sensitive(true);
+    _orc_reader->set_case_sensitive(_case_sensitive);
```

Comment:
`OrcScanner` is used by broker load, not for hive table, are you sure this code works?

Reply:
Hive table is using `hdfs_scanner_orc.cpp`

Reply:
If you want to change for broker load, plz change pr's title & description.

Reply:
pr's title is changed, this pr contain feature's fix for broker load